viz.yaml

```diff
@@ -254,6 +264,7 @@ publish:
       counties_dictionary: "counties_dictionary"
       county_boundaries: "county_boundaries"
       county_centroids: "county_centroids_topojson"
+      wu_data_15_json: "wu_data_15_json"
```

Comment:
and this for sure makes it end up in `target/data`, correct?

Reply:
yup